pkgs/development/ruby-modules/gem/default.nix

```diff
@@ -208,7 +213,13 @@ stdenv.mkDerivation ((builtins.removeAttrs attrs ["source"]) // {
     pushd $out/${ruby.gemPath}
     rm -fv doc/*/*/created.rid || true
     rm -fv {gems/*/ext/*,extensions/*/*/*}/{Makefile,mkmf.log,gem_make.out} || true
-    rm -fvr cache
+
+    # Do not remove the cache as this is needed to make some additional Bundler commands work
+    # such as `bundler install --redownload`
+    # https://github.com/rubygems/rubygems/issues/4088
+    # https://github.com/bundler/bundler/issues/3327
+    # rm -fvr cache
```

Comment:
If you move this comment out of the bash script, it won't cause a mass rebuild and can target the master branch directly.

Reply:
Thanks -- took your suggestion.
I'm curious, how does staging branch make it to master?